packages/client/lib/client/commands-queue.ts

```diff
@@ -1,8 +1,12 @@
-import LinkedList from 'yallist';
-import RedisParser from 'redis-parser';
+import * as LinkedList from 'yallist';
 import { AbortError } from '../errors';
 import { RedisCommandArguments, RedisCommandRawReply } from '../commands';
 
+// We need to use 'require', because it's not possible with Typescript to import
+// classes that are exported as 'module.exports = class`, without esModuleInterop
+// set to true.
+const RedisParser = require('redis-parser');
```

Comment:
I wish there was a way to avoid `require` but I had no luck with this =(

Reply:
Yea, I have found a solution too